tpot/operators/classifiers/base.py

```diff
@@ -57,8 +56,22 @@ def _train_and_predict(self, input_matrix, operator_args):
         clf.fit(self.training_features, self.training_classes)
 
         all_features = np.delete(input_matrix, non_feature_columns, axis=1)
-        input_matrix[:, GUESS_COL] = clf.predict(all_features)
+        if self.clf_eval_func == 'decision_function':
+            if 'decision_function' in dir(clf):
+                input_matrix = np.insert(input_matrix, GUESS_COL, clf.decision_function(all_features).T, axis=1)
```

Comment:
For now, I think we should only support outputs and scorers that output only a single value. Multi-label classification may eventually require a set up that supports multiple outputs, but that's a fair bit down the line.